backend/src/sync/harvest/mod.rs

```diff
@@ -4,9 +4,9 @@ use std::{
 };
 
 use hyper::http::status::StatusCode;
-use tokio_postgres::{GenericClient, types::ToSql};
```

Comment:
Why not use `GenericClient` anymore?

Reply:
Cause we need to call `prepare_cached` which is not part of that trait. It's provided by `deadpool_postgres`.

Reply:
Unfortunate but fair.